crates/ironrdp-web/src/session.rs

```diff
@@ -384,6 +386,7 @@ pub struct Session {
 
 #[wasm_bindgen]
 impl Session {
+    #[allow(unused_assignments)]
```

Comment:
question: What is the reason for suppressing this warning? Can you add a comment along it if it’s really necessary?

Reply:
Sort of janky, but I added a comment describing how to find the explanation alongside the actual false-positive location in the function below: 6e51da5fcce827725ddc5505459c758ab6a5ca02

Here's the lines:

https://github.com/Devolutions/IronRDP/blob/6e51da5fcce827725ddc5505459c758ab6a5ca02/crates/ironrdp-web/src/session.rs#L638-L644

Reply:
update: I don’t get the warning even if I remove the suppressing annotation:

![image](https://github.com/Devolutions/IronRDP/assets/3809077/3f1d74ac-5551-4307-8233-5fa80daca089)

Maybe it’s because of the toolchain version used?

Reply:
Yeah, I had a goofed up local configuration. Fixed this in a888bb1954f03eb505c5844ce0fde8522eadd644